src/main/java/uk/gov/hmcts/reform/sendletter/logging/AppInsights.java

```diff
@@ -34,6 +41,28 @@ public void trackMessageAcknowledgement(java.time.Duration duration, boolean suc
         }
     }
 
+    private double getSecondsSince(ZonedDateTime time) {
+        java.time.Duration sinceTime = java.time.Duration.between(time, ZonedDateTime.now());
```

Comment:
Please can you check if you can use `Chronounit`  `between(...)` method here.https://docs.oracle.com/javase/9/docs/api/java/time/temporal/ChronoUnit.html#between-java.time.temporal.Temporal-java.time.temporal.Temporal-